static/notify/notify.js

```diff
@@ -2,11 +2,12 @@ $(function() {
 
     $('#sendNotificationButton').on('click', function() {
         $.post('/send-notification', {
-           identity: $('#identityInput').val()
+           identity: $('#identityInput').val(),
+           body: "Hello, World!"
```

Comment:
@idelgado I think you should also update [this line](https://github.com/TwilioDevEd/sdk-starter-python/blob/master/app.py#L130) too